packages/core/src/storybook/stand-alone-documentaion/colors/colors.stories.mdx

```diff
@@ -30,27 +30,47 @@ The colors are designed to be clear and accessible. They come in three color the
 
 ### Semantic colors
 
-<SemanticColors />
+<Canvas>
+  <Story name="Semantic Colors">
+    <SemanticColors />
+  </Story>
+</Canvas>
 
 ### Background colors
 
-<BackgroundColors />
+<Canvas>
+  <Story name="Background Colors">
+    <BackgroundColors />
+  </Story>
+</Canvas>
```

Comment:
I think the pattern that you've used here is deprecated. we should use a stories file with `<Canvas of`
not urgent of course, can be fixed later, but let's create a task

we should anyway do the `<Canvas of` instead of `<Story of` refactor in all our stories

Reply:
Yes definitely